packages/nuxt/src/module.ts

```diff
@@ -16,34 +14,25 @@ export default defineNuxtModule<ModuleOptions>({
   },
   defaults: {},
   setup(_moduleOptions, nuxt) {
-    const resolver: Resolver = createResolver(import.meta.url);
-
-    const pathToClientInit = findDefaultSdkInitFile('client');
-
-    if (pathToClientInit) {
-      nuxt.hook('app:templates', nuxtApp => {
-        if (nuxtApp.rootComponent) {
-          try {
-            addImportStatement(nuxtApp.rootComponent, buildSdkInitFileImportSnippet(pathToClientInit));
-          } catch (err) {
-            // eslint-disable-next-line no-console
-            console.error(`[Sentry] Could not add import statement to root component. ${err}`);
-          }
-        }
-      });
+    const moduleDirResolver = createResolver(import.meta.url);
+    const buildDirResolver = createResolver(nuxt.options.buildDir);
+
+    const clientConfigFile = findDefaultSdkInitFile('client');
+
+    if (clientConfigFile) {
+      nuxt.options.alias['#sentry-client-config'] = buildDirResolver.resolve(`/${clientConfigFile}`);
+
+      injectSentryConfigImport('#sentry-client-config');
```

Comment:
m: I have a feeling we're including both templates for client and server in both bundles, since we are not setting the `mode` option.